app/controllers/thesis_controller.rb

```diff
@@ -134,6 +134,30 @@ def process_theses_update
     redirect_to thesis_process_path
   end
 
+  def proquest_export_preview
+    advanced_theses = Thesis.where(publication_status: 'Published').advanced_degree.not_exported_to_proquest
```

Comment:
Yeah, this felt gross even as I wrote it. The job expects three parameters so it can separately update the sets of partial and full harvest thesis records. I don't think a scope would work because the logic is too complex, but a class method is a good idea.